frontend/src/scenes/session-recordings/playlist/sessionRecordingsPlaylistLogic.ts

```diff
@@ -729,40 +662,39 @@ export const sessionRecordingsPlaylistLogic = kea<sessionRecordingsPlaylistLogic
             }
         ] => {
             const params: Params = objectClean({
-                filters: values.customFilters ?? undefined,
+                simpleFilters: values.simpleFilters ?? undefined,
+                advancedFilters: values.advancedFilters ?? undefined,
                 sessionRecordingId: values.selectedRecordingId ?? undefined,
             })
 
-            // We used to have sessionRecordingId in the hash, so we keep it there for backwards compatibility
```

Comment:
Dropping support for this seemed reasonable because it's been around for a while